app/models/proxy_rule.rb

```diff
@@ -3,7 +3,7 @@
 require 'addressable/template'
 
 class ProxyRule < ApplicationRecord
-  acts_as_list scope: :proxy, add_new_at: :bottom
+  acts_as_list scope: %i[owner_id owner_type], add_new_at: :bottom
   scope :ordered, -> { order(position: :asc) }
 
   belongs_to :proxy, touch: true
```

Comment:
I wonder if it should be: :thinking: 
```suggestion
  belongs_to :owner, touch: true
```

Reply:
I don't think it's supported by thinking_sphinx. We have it somewhere else too: https://github.com/3scale/porta/blob/ab2b46785b741acc7714112aeea8eb6a2954cd61/app/models/plan.rb#L18